shoop/front/settings.py

```diff
@@ -63,3 +63,7 @@
 #:
 #: Handled error cases are: 400, 403, 404, and 500
 SHOOP_FRONT_INSTALL_ERROR_HANDLERS = True
+
+#: A list of strings defining which address fields will be required at the form
+#: level in addition to those already required at the model level,
+SHOOP_FRONT_REQUIRED_ADDRESS_FIELDS = []
```

Comment:
The name of this should indicate that these are in addition to required fields in model level.